pallets/loans-ref/src/valuation.rs

```diff
@@ -103,7 +113,7 @@ where
 {
 	pub fn is_valid(&self) -> bool {
 		match self {
-			ValuationMethod::DiscountedCashFlow(dcf) => dcf.discount_rate >= One::one(),
+			ValuationMethod::DiscountedCashFlow(_dcf) => true,
 			ValuationMethod::OutstandingDebt => true,
 		}
 	}
```

Comment:
No longer need this check here (always true). Not sure if removing this

Reply:
I would keep. EVen the check. Checks that interest-accrual is not doing stuff it shouldn't after we use it, right?

Reply:
The check `discount_rate >= 1` needs to be removed since now we expect discount rates less than 1.

Regarding the `interest-accrual`, by now, it's not used with this rate. But when used, the own `interest-accrual` should anyways check its own values 🤷🏻‍♂️

Reply:
This comment no longer apply because now we have an upper limit